packages/admin/src/components/Scenario/CardListContainer/index.tsx

```diff
@@ -0,0 +1,45 @@
+import NoticeCardList from "@admin/components/Scenario/NoticeCardList";
+import { GroupContext } from "@admin/utils/groupContext";
+import { ScrapperConfig, ScrapperContext } from "@admin/utils/scrapperContext";
+import { StatusContext } from "@admin/utils/statusContext";
+import {
+  noticeScenariosMocks,
+  studentRestaurantScenariosMocks,
+  domitoryRestaurantScenariosMocks,
+  colleageScheduleMocks,
+} from "@admin/__mocks__";
+import { useContext } from "react";
+import CardList from "../CardList/CardList";
+import getStyle from "./style";
+
+export default function CardListContainer() {
+  const { group } = useContext(GroupContext);
+  const { scrapper } = useContext(ScrapperContext);
+  const { status } = useContext(StatusContext);
+  const { notice, studentRestaurant, domitoryRestaurant } = ScrapperConfig;
+  const style = getStyle();
+
+  let isNoticeScrapper = false;
+  let data;
+
+  switch (scrapper) {
+    case notice:
+      isNoticeScrapper = true;
+      data = noticeScenariosMocks;
+      break;
+    case studentRestaurant:
+      data = studentRestaurantScenariosMocks;
+      break;
+    case domitoryRestaurant:
+      data = domitoryRestaurantScenariosMocks;
+      break;
+    default:
+      data = colleageScheduleMocks;
+  }
```

Comment:
- `isNoticeScrapper`와 `data`는 useState로 관리하고 값을 `set`하는 로직은 의존성 배열에 적절한 값을 넣어서`useEffect`를 
활용하면 좋을것 같습니다.
- `data`라는 변수명은 최대한 지양하는게 좋습니다.. 너무 포괄적인 명칭이라 어렵겠지만 최대한 적절한 변수명을 지어주세요 ㅜㅜ
- 개인적으로 `swtich~case`문을 별로 좋아하지않아서 `if` 문으로 `early return` 방식을 추천드립니다

예시
```suggestion
  const [scenario,setScenario] = useState(적절한 초기값)
  const [isNoticeScraper, setIsNoticeScraper] = useState(false)
  
  useEffect(()=>{
  
  if(scraper === ScrapperConfig.notice){
       setScenario(noticeScenariosMocks)
       setIsNoticeScraper(true)
       return
   }
      ....
  },scraper)

`````

Reply:
아아 `isNoticeScraper`를 useState로 관리할 생각을 못 했네요ㅠㅠ 그리고 scrapper라고 잘못 쓴 부분도 모두 고쳐야겠습니다 🙏

Reply:
넵 ㅎㅎ 그리고 한가지 더 말씀드리면 변수를 불변으로 관리하는것이 사이드이펙트의 발생률을 낮춰주기 때문에 `let` 보다는 `const` 사용을 지향하도록 하는게 좋을것 같습니다 물론 가끔 필요하면 `let`을 써야하긴 하겠지만 한번 더 고민해서  변수를 불변하게 관리할 수 있는 방법은 없는지 `const`로 선언해서 사용할순없는지 한번 더 생각해보면 좋을것 같아요~

Reply:
아하 이 코드에서 `data` 변수는 핵심 데이터인 만큼 상수로 다룰 수 있는 방법을 더 고민해야 할 것 같습니다!